gidgethub/httpx.py

```diff
@@ -1,13 +1,16 @@
 import asyncio
 from typing import Mapping, Tuple, Any
 
-import httpx
+try:
+    from httpx import AsyncClient as Client  # type: ignore
+except ImportError:
+    from httpx import Client
```

Comment:
I assume the old sync class was (unfortunately) named `Client`?

Reply:
I think so. IIUC there is no longer a sync client - httpx is async only (for now). There may be plans afoot to reintroduce a sync client but I'm not fully across all of that.

With httpx & hip, async http libraries are a rapidly evolving field in Python!